Justfile

```diff
@@ -139,6 +139,7 @@ run-graph-builder:
 		verbosity = "vvv"
 
 		[service]
+		scrape_timeout_secs = 300
```

Comment:
Perhaps its worth excluding initial scrape and setting a 30 sec timeout for subsequent scrapes?

Readiness check would kill the container and roll back the deployment if the initial scrape doesn't pass in 3 + 10*3 + 3*3 = 42 seconds in any case

Reply:
It's tempting but I don't think we really need to set this so tightly. This timeout is meant as an emergency break in case the scraping gets stuck. Subsequent to this PR we can work on individual request timeouts.